### ISSUE TITLE
Bottom right box confusing at times

### ISSUE THREAD

**Issue by philippe-hawi** [Dijji/CustomWindowsProperties#7]
If a property is selected in the bottom left box, and a new property is being edited in the top right box, the information displayed in the bottom right box can be confusing.
If the focus is in one the fields in the top right box, a help message for that field is displayed in the bottom box. If the focus is not in any field, the bottom box reverts to displaying the info of the selected property from the bottom left box.

**Comment by Dijji** [comment]
This is true. But what do you think it ought to be? The idea is that you might be using the information in the bottom right box about how system properties are configured to see what you might set in the top right.

Having got this far, I also want to thank you for all the excellent feedback that you have been giving. Perfect for making the program stronger. Dijji

**Comment by philippe-hawi** [comment]
> This is true. But what do you think it ought to be? The idea is that you might be using the information in the bottom right box about how system properties are configured to see what you might set in the top right.

I completely agree with this. That is why I would've much preferred to keep the bottom right box exclusive to the selected property on the left. Many times, while creating a custom property, I would have to click in a white space to bring up the info of the selected property that I was using as reference. Since it seems that the only info displayed in the bottom right box that is related to the property being created is help/documentation, perhaps it's better to move that help to somewhere else. Maybe create a new box that shows a preview of the info of the property being created? Or split the bottom right box and show, side by side (maybe as a table with alternating row colors), the info of both the selected existing property (if any is selected) and the new property being created. I found myself looking for that feature in order to easily compare the fields of the my new property with the reference property.
Again, will give it more thought and get back to you.

**Comment by Dijji** [comment]
I'm playing around at the moment with adding two more panels on the right of the window. 

At the top right goes the help text - no more hiding the installed property's details.

At the bottom right goes a comparison panel, which shows you the differences between the property in the editor and either its saved original or the selected installed property, depending which you choose using radio buttons at the top of the panel.

I am also working on adding context menus, though I will leave the buttons in place for the moment, to see if they still add any value, with the exception of the Export button, which never did make that much sense.

Does this sound in the right direction? I hope you still recognise at least some of your ideas. Also, if you have thought further, do let me know the results.

Dijji

**Comment by philippe-hawi** [comment]
Most of what you're doing sounds very much like what I had in mind. I think the 2 extra panels are the way to go. Very good ideas. I'll wait until I try it and then we'll reevaluate.

**Comment by Dijji** [comment]
Okay, the new version, 0.3 has now been posted and is ready for you to try it. I think that the line discussed above has worked out quite well, but I would very much like to hear what you think. Note that the README has been updated to describe the new functionality.
